docs/Tutorial_MPDist.ipynb

```diff
@@ -0,0 +1,309 @@
+{
```

Comment:
<p>I think it's <em>hierarchical </em>clustering, not entirely sure though so double check this.</p><p> </p><p>You can capitalize "Euclidean".</p><p> </p><p>Replace "MPdist" with "MPDist" in the title (or change everywhere else to "MPdist", whichever is more convenient).</p><p> </p><blockquote>More concretely. given a threshold p and window length L, the MPdist between two time series is the smallest distance such that p * 100% of their windows of size L have a lower euclidean distance to some window of equal size in the other .&nbsp;</blockquote><p> </p><p>Maybe you could rephrase this, I wasn't sure what this meant on first glance. Maybe say p% instead of p * 100%? Maybe rephrase to something like: "... the MPDist between two time series is the smallest pairwise Euclidean distance such that it is higher in value than p% of pairwise Euclidean distances", or some variation of this. It's a tough definition to summarize...</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/433/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>To be consistent with the paper, let's use "MPdist" everywhere please</p>

Reply:
<p>Fixed the spelling and captilization errors - thank you for pointing them out! As for rephrasing the paragraph, I tried doing a better job by referencing join matrix profile - do you think I should define that as well or maybe link back to the paper?</p>